corehq/apps/sms/change_publishers.py

```diff
@@ -1,10 +1,20 @@
 from corehq.apps.change_feed import topics
```

Comment:
was this necessary or just seemed good to do? seems fine, just wondering if there was an issue with publishing in general